common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -419,7 +419,8 @@ def run_imports(self):
                 dest_id, runtime = self.get_dest_id(courselike_key)
             except DuplicateCourseError:
                 continue
-            with self.store.bulk_operations(dest_id):
+            #with self.store.bulk_operations(dest_id):
+            if True:
                 source_courselike, courselike, data_path = self.get_courselike(courselike_key, runtime, dest_id)
```

Comment:
Yeah, let's talk. I'd love to understand why using `bulk_operations` breaks things (and figure out how to fix it).